docs/source/traits_user_manual/advanced.rst

```diff
@@ -38,15 +38,23 @@ attribute, with a name based on the name of the trait attribute:
 
 .. method:: _name_default()
 
-This method initializes the *name* trait attribute, returning its initial value.
-The method overrides any default value specified in the trait definition. The
-initializer is called when:
+This method returns the default value for the *name* trait attribute and it
+overrides any default value specified in the trait definition.
+
+Similar to static default values, default values defined dynamically are
+considered to have existed **prior to** setting object state during
+initialization. For performance purposes, a default initializer is called when:
```

Comment:
I think I came across the term "default initializer" somewhere, but I am not sure if this is necessarily the right name for the method. "default constructor"? "default handler"?

Reply:
Maybe `default initializer method`? I think I usually just refer to these things as "traits default methods", but that's a bit too vague.

Reply:
SGTM. I will add that.